### ISSUE TITLE
Sprint 4 – Scrum Meeting / Retrospective (25-12-2016)

### ISSUE THREAD

**Issue by lkc442012** [mewind/RSDSupermarketPOS#27]
Merry Christmas to everyone! Today is the last day of Sprint 4, which is also our last sprint in this project. After today, our project can be consider done. Good job guys!

**Comment by huylin25** [comment]
Wow!

**Comment by lkc442012** [comment]
Until now, the project is built successfully in the Team Service.

**Comment by mewind** [comment]
Thats good!

**Comment by lkc442012** [comment]
So did you guys facing any obstacles in this week? Lets discuss it together to review those challenges.

**Comment by huylin25** [comment]
Actually I have experienced some difficulties when develop the generation of sales report. I cannot get the output that I want by using SQL command.

**Comment by juzplayplay** [comment]
I know the problem that you was facing. This is because I also cannot do it in my exception and summary report.

**Comment by huylin25** [comment]
Yes. I do not understand about the syntax of the SQL command to display the view by combining more than one column in the database. This cause me to stuck in that part for some times.

**Comment by lkc442012** [comment]
So how do you solve it?

**Comment by huylin25** [comment]
Chun Xiong and I have try using several methods to write the SQL command and finally we found the correct way to display the output that we want.

**Comment by juzplayplay** [comment]
You are right. After searching for some information using Google, we have a clear idea that what we should include in the command.

**Comment by lkc442012** [comment]
Alright. So that is the main challenge in this week.

**Comment by lkc442012** [comment]
In conclusion, we have done the following part in this week: 
1) Generation of sales report - Huy Lin
2) Generation of exception report - Chun Xiong
3) Generation of summary report - Chun Xiong
4) Activity log - Sin Nee
5) Scrum Master Report -  Sin Nee
6) Unit testing -  Sin Nee

**Comment by mewind** [comment]
So the project is finished?

**Comment by lkc442012** [comment]
Yes, there is nothing that is needed to change for the system. I think thats all for today's meeting. Thank you everyone for your effort in this project!

**Comment by mewind** [comment]
You are welcome!
